### PR TITLE
chore(ci): fix k8s jobs, better shebang 

### PR DESCRIPTION
**Pull request by zdrapela**
## Description

- Fix k8s CI job configurations after https://github.com/janus-idp/backstage-showcase/pull/2032 
- Use `#!/bin/bash` shebang in CI files. This is appropriate for how the scripts are written and is recommended by https://github.com/koalaman/shellcheck. Also incorporate more improvements from ShellCheck.

## Which issue(s) does this PR fix

- Fixes [RHIDP-5214](https://issues.redhat.com/browse/RHIDP-5214)

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

**Comment by zdrapela** [comment]
After recently merged PRs the k8s jobs are broken again.

File: .ibm/pipelines/utils.sh [hunk 1]
```diff
@@ -549,7 +549,9 @@ run_tests() {
   cp -a "/tmp/${LOGFILE}.html" "${ARTIFACT_DIR}/${project}"
   cp -a /tmp/backstage-showcase/e2e-tests/playwright-report/* "${ARTIFACT_DIR}/${project}"
 
-  droute_send "${release_name}" "${project}"
+  if [[ "${OPENSHIFT_CI}" == "true" ]]; then
```
Comment (subhashkhileri) [inline .ibm/pipelines/utils.sh:552]: just a suggestion, this method is becoming big and big. does this makes sense to move this condition inside the droute_send() and something like
if [[ "${OPENSHIFT_CI}" != "true" ]]; then
    return 0
fi

Comment (zdrapela) [reply .ibm/pipelines/utils.sh:552]: Yes, we can do that 👍

**Comment by zdrapela** [comment]
/override PR / Test with Node.js 20

**Pull request merged**